src/h5web/visualizations/containers/hooks.ts

```diff
@@ -16,5 +16,5 @@ export function useDatasetValues(
   const { getValues } = useContext(ProviderContext);
   const [valuesReader] = useAsyncResource(getValues, datasets);
 
-  return valuesReader() || {};
+  return valuesReader();
```

Comment:
`valuesReader()` either returns the fetched data or throws if they're still being fetched. So we don't need a fallback, and we don't need an early return in `useNxData` after all.